Source/WebKit/Shared/RemoteLayerTree/RemoteLayerBackingStoreCollection.mm

```diff
@@ -101,9 +100,8 @@
             });
 
             backingStoreList.append(*remoteBackingStore);
-        }
-
-        backingStore->prepareToDisplay();
+        } else
+            backingStore->prepareToDisplay();
```

Comment:
I think we could still call this unconditionally on both types (maybe first?), and then just have the branch for appending to the serialization lists.

Reply:
Done